R/rd-include-rmd.R

```diff
@@ -0,0 +1,61 @@
+
+block_include_rmd <- function(tag, block, base_path) {
+  rmd <- tag$val
+  stopifnot(is.character(rmd), length(rmd) == 1, !is.na(rmd))
+  if (!requireNamespace("rmarkdown", quietly = TRUE)) {
+    stop("@includeRmd requires the rmarkdown package")
```

Comment:
```suggestion
    abort("@includeRmd requires the rmarkdown package")
```

(For consistency this should probably be with a warning that gives the tag location, but this is likely to be a rare error so it's not worth it)

Reply:
OK, made it into a warning.